MD-Verlet-Integrator/md_prog.py

```diff
@@ -18,16 +35,14 @@
 max_md_step = 10000                 # Number of MD iterations
 veloc0 = np.zeros((2,3))            # Numpy array (natoms,3) with inital velocities
 trajec = True                       # Boolean: Save all trajectories in a single xyz file (md_trajectories)  for visualization
-grad_method = 'hf/3-21g'            # QC method (with basis set) for energy and gradient
+grad_method = 'hf/3-21g'            # Method (QC with basis set) for energy and gradient
+int_alg = 'veloc_verlet'            # Algorithm to use as integrator
 opt    = False                      # Optimize geometry using F=ma
 
-# Molecular Dynamics Function
-def molec_dyn(timestep,max_md_step,veloc0,grad_method,trajec = True,opt=False):
+# Molecular Dynamics Main Program
     
     #Get initial positions,velocities, accelerations and forces
-    #timestep *= fs_timeau
-    energy,wfn = psi4.energy(grad_method,return_wfn=True)
-    forces = -np.asarray(psi4.gradient(grad_method,ref_wfn=wfn))
+    energy,forces = md_helper.get_forces(grad_method)
```

Comment:
Since these lines (45-79) are no longer within the `molec_dyn()` function, they no longer need to be indented